src/test/resources/files/InventoryUUIDs.cql

```diff
@@ -0,0 +1 @@
+(source=="FOLIO") sortby title
```

Comment:
This is a duplicate file, same as above

Reply:
it's because for DataExportTest and FileUploadServiceTest we use different directories to get files, it's easier to have two the same files than change this mechanism